Safe mode (#188)
omichel/webots: docs/guide/starting-webots.md

The critical Ogre log messages are redirected there in any case.
If the `--stdout` option is enabled then the uncritical Ogre log messages are redirected to the *stdout* stream of the Webots executable instead.
Similarily, the `--stderr` option is redirecting the Ogre critical messages to the *stderr* stream.

### Safe Mode

> It may happen that Webots can not start because it is blocked on a world causing a Webots or OpenGL crash.

Reviewer: ```suggestion
It may happen that Webots cannot start because it is blocked on a world causing a Webots or OpenGL crash.
```

In this case, Webots can be started in safe mode.

> The safe mode forces Webots to be started on an empty world, reduce all the OpenGL options, and store those preferences.

Reviewer 2: ```suggestion
The safe mode forces Webots to start with an empty world, reduces all the OpenGL options, and stores those preferences.
```

To do this, simply set the environment variable `WEBOTS_SAFE_MODE` in the environment running Webots.

> Once successfully started this way, you must unset this environment variable, open again your world and increase the OpenGL preferences.

Reviewer 2: A link to the opengl preference doc would be welcome
